src/mem/metaslab.h

```diff
@@ -3,30 +3,13 @@
 #include "../ds/cdllist.h"
 #include "../ds/dllist.h"
 #include "../ds/helpers.h"
+#include "freelist.h"
 #include "sizeclass.h"
 
 namespace snmalloc
 {
   class Slab;
 
-  /**
-   * Free objects within each slab point directly to the next (contrast
-   * SlabLink, which chain different Slabs of the same sizeclass together).
-   */
-  struct SlabNext
```

Comment:
Hm.  Sorry to have churned the code base by introducing this only for it to be removed.

Reply:
Don't worry, it was actually useful the churn gave me the places I need to change it for this.